src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -102,6 +103,22 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
             _call_az_command(create_repair_vm_command, secure_params=[repair_password, repair_username])
             logger.info('Attaching copied disk to repair VM...')
             _call_az_command(attach_disk_command)
+
+            # Install extension in case of single pass encrypted VM
+            disk_id = source_vm.storage_profile.os_disk.managed_disk.id
+
+            disk_settings_command = 'az disk show --ids {ids} -o json' \
+                                    .format(ids=disk_id)
+            settings = _call_az_command(disk_settings_command)
+            settings1 =  json.loads(settings)
+            settings2 = (settings1["encryptionSettingsCollection"]["encryptionSettings"])
+            key_vault = ( settings2[0]['keyEncryptionKey']['sourceVault']['id'] )
+            key_encryption_url = ( settings2[0]['keyEncryptionKey']['keyUrl'] )
```

Comment:
Is there any particular reason to add () to the expression?
And the format is not unified. Sometimes there is a space, sometimes there is not space.